content/applications/productivity/discuss/chatter.rst

```diff
@@ -0,0 +1,416 @@
+
+.. image:: chatter/send-message-followers.png
+   :align: center
+   :alt: A chatter composer preparing to send a message to the followers of a CRM opportunity and
+         the customer listed on the opportunity record.
+
+Expand full composer
+--------------------
+
+The chatter composer can be expanded to a larger pop-up window, allowing for additional
+customizations.
+
+To open the full composer, click the :guilabel:`expand` icon in the bottom-right of the composer
+window.
+
+.. figure:: chatter/chatter-expand-icon.png
+   :align: center
+   :alt: A chatter composer with emphasis on the expand icon.
+
+   The expand icon in a chatter composer.
+
+Doing this opens a :guilabel:`Compose Email` pop-up window. Confirm or edit the intended
+:guilabel:`Recipients` of the message, or add additional recipients. The :guilabel:`Subject` field
+auto-populates based on the title of the record, though it can be edited if desired.
+
+To use an :doc:`email template <../../general/companies/email_template/>` for the message, select it
+from the :guilabel:`Load template` drop-down.
+
+.. note::
+   The number and type of templates available vary based on the record the message is created from.
+
+Click :guilabel:`📎 Attachments` to add any files to the message, then click :guilabel:`Send`.
+
+.. image:: chatter/chatter-full-composer.png
+   :align: center
+   :alt: The expanded full chatter composer in the CRM application.
+
+Edit sent messages
+------------------
+
+Messages can be edited after they are sent, to fix typos, correct mistakes, or add missing
+information.
+
+.. note::
+   When messages are edited after they have been sent, an updated message is **not** sent to the
+   recipient.
+
+To edit a sent message, click the :guilabel:`... (ellipsis)` icon menu to the right of the message.
+Then, select :guilabel:`Edit`. Make any necessary adjustments to the message.
+
+.. image:: chatter/chatter-edit.png
+   :align: center
+   :alt: The edit message option in a chatter thread.
+
+To save the changes, press :command:`Ctrl + Enter`. To discard the changes, press :command:`Escape`.
+
+.. important::
+   Users with Admin-level access right can edit any sent messages. Users without Admin rights can
+   **only** edit messages they created.
+
+.. _discuss/search-messages:
+
+Search messages
+===============
+
+Chatter threads can become long after a while, because of all the information they contain. To make
+it easier to find a specific entry, users can search the text of messages and notes for specific
+keywords.
+
+Select a record with a chatter thread. For example, to search a *CRM* opportunity, navigate to
+:menuselection:`CRM app --> Sales --> My Pipeline` and click on the Kanban card of an opportunity to
+open it. Then, at the top-right above the chatter composer, click the :guilabel:`🔍 (magnifying
+glass)` icon to open the search bar.
+
+Enter a keyword or phrase into the search bar, then hit enter, or click the :guilabel:`🔍
```

Comment:
```suggestion
Enter a keyword or phrase into the search bar, then hit :command:`Enter`, or click the :guilabel:`🔍
```
would this fall in line with the other instances in which you used :command:? Might be nice to use here, as well, for uniformity... your call